CRM/Campaign/BAO/Campaign.php

```diff
@@ -399,6 +399,7 @@ public static function getCampaignSummary($params = [], $onlyCount = FALSE) {
       if ($orderOnCampaignTable) {
         $orderByClause = "ORDER BY campaign.{$sortParams['sort']} {$sortParams['sortOrder']}";
       }
+      $orderByClause .= ", campaign.id {$sortParams['sortOrder']}";
```

Comment:
On line 365 above `$orderByClause` is declared as NULL, so this concatenation would need to be conditional on it actually containing some value.